src/main/kotlin/no/nav/syfo/dialogmotestatusendring/DialogmoteStatusendringService.kt

```diff
@@ -1,27 +1,31 @@
 package no.nav.syfo.dialogmotestatusendring
 
 import no.nav.syfo.dialogmotestatusendring.domain.DialogmoteStatusendring
-import no.nav.syfo.dialogmotestatusendring.domain.DialogmoteStatusendringType
-import no.nav.syfo.dialogmotestatusendring.kafka.*
-import no.nav.syfo.personoppgave.behandleOppgave
+import no.nav.syfo.dialogmotestatusendring.domain.happenedAfter
+import no.nav.syfo.dialogmotestatusendring.kafka.log
+import no.nav.syfo.personoppgave.*
+import no.nav.syfo.personoppgave.domain.toPersonOppgave
 import java.sql.Connection
+import java.util.*
 
 fun processDialogmoteStatusendring(
     connection: Connection,
     statusendring: DialogmoteStatusendring,
 ) {
     log.info("Received statusendring of type ${statusendring.type} and uuid ${statusendring.dialogmoteUuid}")
 
-    if (statusendring.type != DialogmoteStatusendringType.INNKALT) {
-        val success = connection.behandleOppgave(
-            statusendring.dialogmoteUuid,
-            statusendring.veilederIdent,
-        )
-
-        if (success) {
-            // TODO: add oppgavebehandlet of dialogmotesvar_behandlet to personoppgavehendelse topic
-        } else {
-            log.info("No personoppgave to update for ${statusendring.dialogmoteUuid}")
+    val ppersonOppgave = connection.getPersonOppgaveByReferanseUuid(statusendring.dialogmoteUuid)
+    if (ppersonOppgave == null) {
+        val personoppgaveUuid = UUID.randomUUID()
```

Comment:
Kan evt generere denne uuid-en i `createBehandletPersonoppgave` siden den ikke brukes til annet her.

Reply:
Det var to grunner:
1. Da slipper man å spørre databasen hvis man trenger å vite dette senere (som erstatning for at vi returnerte den i en Pair).
2. UUIDen kan jo sendes rundt om kring, og da er det vel ikke databaselogikk, og burde følgelig ikke ligge i databaselaget. 

Men vi er jo ikke nødt til å ha det sånn da. 🤷